DQM/SiPixelHeterogeneous/plugins/SiPixelCompareTrackSoAAlpaka.cc

```diff
@@ -210,22 +210,22 @@ void SiPixelCompareTrackSoAAlpaka<T>::analyze(const edm::Event& iEvent, const ed
     nLooseAndAboveTracksHost_matchedDevice++;
 
     hchi2_->Fill(tsoaHost.view()[it].chi2(), tsoaDevice.view()[closestTkidx].chi2());
-    hCharge_->Fill(reco::charge<T>(tsoaHost.view(), it), reco::charge<T>(tsoaDevice.view(), closestTkidx));
+    hCharge_->Fill(reco::charge(tsoaHost.view(), it), reco::charge(tsoaDevice.view(), closestTkidx));
     hnHits_->Fill(helper::nHits(tsoaHost.view(), it), helper::nHits(tsoaDevice.view(), closestTkidx));
     hnLayers_->Fill(tsoaHost.view()[it].nLayers(), tsoaDevice.view()[closestTkidx].nLayers());
     hpt_->Fill(tsoaHost.view()[it].pt(), tsoaDevice.view()[closestTkidx].pt());
     hptLogLog_->Fill(tsoaHost.view()[it].pt(), tsoaDevice.view()[closestTkidx].pt());
     heta_->Fill(etaHost, tsoaDevice.view()[closestTkidx].eta());
-    hphi_->Fill(phiHost, helper::phi(tsoaDevice.view(), closestTkidx));
-    hz_->Fill(zipHost, helper::zip(tsoaDevice.view(), closestTkidx));
-    htip_->Fill(tipHost, helper::tip(tsoaDevice.view(), closestTkidx));
+    hphi_->Fill(phiHost, reco::phi(tsoaDevice.view(), closestTkidx));
+    hz_->Fill(zipHost, reco::zip(tsoaDevice.view(), closestTkidx));
+    htip_->Fill(tipHost, reco::tip(tsoaDevice.view(), closestTkidx));
```

Comment:
True (though I didn't check that it works).

Personally, I would prefer to keep the `reco::` namespace -- though I admit it may not help too much to understand where these are defined :-/

Reply:
(for the records it works)